namui-cli/electron/src/main.js

```diff
@@ -1,17 +1,42 @@
-const { app, BrowserWindow, ipcMain } = require("electron");
+const { app, BrowserWindow, ipcMain, shell } = require("electron");
 const isDev = require("electron-is-dev");
+const { existsSync, readFileSync } = require("fs");
 const path = require("path");
 
 const config = getConfig();
+const gotTheLock = app.requestSingleInstanceLock();
+
+let mainWindow;
```

Comment:
[Support deep link](https://github.com/NamseEnt/namseent/pull/295/commits/39ec3672249ea417cd309bea6129d73a43862fa5)
commit on this branch is outdated. I will fix it